services/api.py

```diff
@@ -32,6 +33,7 @@
 from rest_framework import renderers
 from django.template.loader import render_to_string
 from django.utils.module_loading import import_string
+from pydantic import ValidationError as pydantic_ValidationError
```

Comment:
I would just say

    import pydantic

here, and later use pydantic.ValidationError.